traffic_ops/app/db/admin.pl

```diff
@@ -64,7 +64,7 @@
 	. "dropdb  - Execute db 'dropdb' on the database for the current environment.\n"
```

Comment:
Since this script is deprecated I don't think we should be adding new functionality to it. I think the new functionality should only be added to admin.go unless there is a very good reason it also needs to be in admin.pl. If there is a very good reason for it, then admin.pl should just shell out to the admin binary so that the checks aren't duplicated.

Reply:
If we want to take this even further I was just thinking of having the admin.pl script shell out all of its individual commands to the binary. So the admin.pl script would just be a tiny perl script that basically only does something like this:
```
my $new_admin = dirname(__FILE__) + "/admin";
shift @ARGV;
my $args = join " ", @ARGV;
system("$new_admin $args");
```
We could probably even just convert it into a bash script still named admin.pl, but I'd be concerned about the possible confusion with that.

Reply:
> having the admin.pl script shell out all of its individual commands to the binary

+1

>We could probably even just convert it into a bash script still named admin.pl

-1. Perl is required by the Linux Standard Base (LSB), so there's not really any downside of a basic Perl script that doesn't require any dependencies or external modules (i.e. the absurd install time of TO/CPAN).

Reply:
Took me a few tries, but I think I got it right now.

Reply:
I agree, no reason to cause such confusion.
I like the idea of using the binary for everything, but I don't think it belongs in the same commit as added functionality.

Reply:
> I like the idea of using the binary for everything, but I don't think it belongs in the same commit as added functionality.
+1, would be better to review and merge the PR that makes admin.pl shell everything out to the binary first. Then the new functionality can just be reviewed in admin.go.

Reply:
#3186

Reply:
rebased on master after #3186 was merged - this should be good to go now.